atomata/capstone_WebClient
# Firefox UI Fixes V2

### Summary 
Fixes scrollbars and text box
### Details
- Fixed action sequence, apparatus info, skybox info, text preview, and workbench scrollbars
- Fixed text editor resizing
### How to test?
1. Verify that all scrollbars are stylized
2. Verify that text editor can be resized
3. Test on both Firefox and Chrome
### Checks
